model/src/main/scala/org/broadinstitute/dsde/rawls/model/JsonSupport.scala

```diff
@@ -95,7 +96,7 @@ trait TypedAttributeListSerializer extends AttributeListSerializer {
  * NOTE: When subclassing this, you may want to define your own implicit val attributeFormat with an AttributeListSerializer mixin
  * if you want e.g. the plain old JSON array attribute list serialization strategy
  */
-class JsonSupport extends DefaultJsonProtocol {
```

Comment:
Hussein's investigation into spray json implicit errors rescued me from this evil line

i was getting all sorts of spray json implicit errors even when importing the unmarshallers right above where they were being used. removing the `extends DefaultJsonProtocol` here fixed my problem but required me to import `DefaultJsonProtocol._` wherever we used spray json implicit unmarhsallers